packages/playground/src/components/DemoFrame.tsx

```diff
@@ -74,7 +64,7 @@ export default function DemoFrame(props: DemoFrameProps) {
     );
     setContainer(instanceRef.current.contentDocument.body);
     setWindow(() => instanceRef.current.contentWindow);
-  }, []);
+  }, [instanceRef]);
```

Comment:
I don't think this will do anything because the `ref` will never change

Reply:
technically the `instanceRef.current` can be changed during the mount/unmount... it is technically correct to have it there

Reply:
But even when `current` changes, `instanceRef` will be the same object

Reply:
True... And having it as a dependency doesn't hurt and is technically correct